openapi.yaml

```diff
@@ -1,279 +1,678 @@
+        fax:
+          type: string
+          example: 541-111-1111
+        thumbnails:
+          type: array
+          items:
             type: string
-            description: Name of pet
-            example: Hedwig
-          owner:
+            format: url
+            example: https://example.oregonstate.edu/image.jpg
+        images:
+          type: array
+          items:
             type: string
-            description: Name of owner
-            example: Harry Potter
-          species:
+            format: url
+            example: https://example.oregonstate.edu/image.jpg
+        departments:
+          type: array
+          items:
             type: string
-            description: Species of pet
-            example: Owl
-  PetResult:
-    properties:
-      links:
-        $ref: '#/definitions/SelfLink'
-      data:
-        $ref: '#/definitions/PetResource'
-  PetsResult:
-    properties:
-      links:
-        allOf:
-          - $ref: '#/definitions/SelfLink'
-          - $ref: '#/definitions/PaginationLinks'
-      meta:
-        $ref: '#/definitions/Meta'
-      data:
-        type: array
-        items:
-          $ref: '#/definitions/PetResource'
-  PetPostBody:
-    type: object
-    properties:
-      data:
-        type: object
-        properties:
-          type:
-            $ref: '#/definitions/PetType'
-          attributes:
-            type: object
-            properties:
-              name:
-                $ref: '#/definitions/PetResource/properties/attributes/properties/name'
-              owner:
-                $ref: '#/definitions/PetResource/properties/attributes/properties/owner'
-              species:
-                $ref: '#/definitions/PetResource/properties/attributes/properties/species'
-            required:
-              - name
-              - owner
-              - species
-            additionalProperties: false
-        required:
-          - type
-          - attributes
-        additionalProperties: false
-    required:
-     - data
-    additionalProperties: false
-  PetType:
-    type: string
-    enum: [pet]
-  ErrorObject:
-    properties:
-      status:
-        type: string
-        description: HTTP status code
-        example: '123'
-      title:
-        type: string
-        description: A short, user readable summary of the error
-        example: Not Found
-      code:
-        type: string
-        description: An application-specific error code
-        example: '1234'
-      detail:
-        type: string
-        description: A long description of the error that may contain instance-specific details
-      links:
-        properties:
-          about:
+            example: College of Engineering
+        website:
+          type: string
+          format: url
+          example: https://example.oregonstate.edu
+        sqft:
+          type: string
+        calendar:
+          type: string
+        campus:
+          type: string
+          example: Corvallis
```

Comment:
Your parameter doesn't have this option. We should make them be consistent (case sensitive). You also need an enum here.